### DISCUSSION TITLE
hsr motor board broken

### DISCUSSION THREAD

**Discussion by Yoo-Youngjae** [HumanSupportRobot/QandA#42] [Q&A]
Hi. It's been a while since I've been writing again.

HSR doesn't wake up after the orange light and saying error message "failed to launch.  retrying."

It seems that the base_r_drive_wheel_joint's motorboard is broken, just like the hand moterboard before us.

When we log into the robot using administrator privileges, then set it to not automatically start up using the following command:

$ sudo systemctl stop docker.hsrb.robot.service

Release the stop button, then, after waiting about 10 seconds

we execute the following "exxx_hash" command:
$ exxx_hash /dev/ttyCTI2 12

And connection timed out is coming out.

And I also checked that the fuse on the motor board at number 12 base_r_drive_wheel_joint's was not blown.

Can I get the motor board again?

thank you.

<img width="689" alt="KakaoTalk_Photo_2023-02-25-01-08-49" src="https://user-images.githubusercontent.com/44061214/221228328-e297299b-4752-4793-8b18-a82bf3c351d4.png">

**Comment by hsr-admin** [comment]
Please wait a moment while we send you the motor board for the replacement base_r_drive_wheel_joint's #12.

**Reply by Yoo-Youngjae** [reply]
Thank you. I'll email you our address.

And this is the second motor board failure. Can I get other motor boards together?
I'm going to go to a competition with the HSR soon, but I'm very worried that another motor board will break down again on the site.

Thank you.
